### ISSUE TITLE
Recision-Recall plots

### ISSUE THREAD

**Issue by helpmefindaname** [helpmefindaname/ner-eval-dashboard#6]
PR-plots are usually done for classification tasks, however it is also possible to also does this for named entities:
On the test set, compute the scores of the labels:
On a linear head, this is very simple: compute the min/max/mean/multiplied confidence of the tags corresponding to a span.
On a CRF, this is more complicated: one way would be to use the forward-backward algorithm to compute tag scores, or reduce the sequence to only the entity and run the pre-crf scores through the viterbi algorithm to compute the log-likelihood of the span.

Idealy, the handling is a hyperparameter, such that different MAP scores can be provided and suggest the best way of computing the score based on the validation set.
